docs/pbc/all/tax-management/third-party-integrations/avalara.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
This page must be inside a folder `docs/pbc/all/tax-management/third-party-integrations/avalara/avalara.md`